lib/migrator.ex

```diff
@@ -1,8 +1,11 @@
-defmodule Neo4Ecto.Migrator do
+defmodule Ecto.Neo4Ecto.Migrator do
   @moduledoc """
   Handles migrations modules loading and version control
   """
-  alias Neo4Ecto.Migration.Runner
+
+  alias Bolt.Sips
+  alias Ecto.Adapters.Neo4Ecto
```

Comment:
Preciosismo aqui hue só uma sugestão mesmo (famoso, só pra comentar algo nesse PR, brinks)

```suggestion
  alias Ecto.{Adapters.Neo4Ecto, Neo4Ecto.Migration.Runner}
```